### ISSUE TITLE
Events: Derive data from meetup.com api

### ISSUE THREAD

**Issue by klequis** [trivalleycoders-org/trivalleycoders#34]
Events are currently coming from the database.
Once this task is complete and merged into master the current MongoDB data should be removed.
- Call meetup.com api to get events for our group
- Create a folder & file /client/externalApi/index.js
- Do api call from this new module using fetch (you may need to import isomorphic-fetch
- Modify existing actions to work with new api
* It may be that the component needs not modification - at least it is possible

**Comment by LauraSilvey** [comment]
I would like to work on this if possible.

-Laura

**Comment by LauraSilvey** [comment]
Hi Karl,

Should the folder and file path be /client/**src**/externalApi/index.js?  This is similar to the api folder and file that is already in the project structure.

**Comment by klequis** [comment]
Sorry Laura. So busy on styling I have not been watching issues.

Put your fetch code in api/index.js. I'm not sure if express will intercept
or not. It may do and likely will fail, maybe silently

**Comment by LauraSilvey** [comment]
I've added the meetup api fetch code to api/index.js.  Will need help hooking up to express.

**Comment by klequis** [comment]
First question we need to answer is if it should go through express or not. Some googling should tell us. If not, we will try it either way.
